src/dashboard/components/DonationSummary.vue

```diff
@@ -84,7 +90,8 @@ export default {
   computed: {
     ...mapState('settings', ['budget_per_month']),
     total() {
-      return _.sumBy(this.distribution, 'funds');
+      return this.budget_per_month;
+      // return _.sumBy(this.distribution, 'funds');
```

Comment:
remove total and replace with budget_per_month wherever it is used

Reply:
changed it back to sum over distribution